client/cache/read_through.go

```diff
@@ -0,0 +1,67 @@
+package cache
+
+import (
+	"context"
+	"fmt"
+	"github.com/beego/beego/v2/core/berror"
+	"strings"
+	"time"
+)
+
+// readThroughCache is a decorator
+// add the read through function to the original Cache function
+type readThroughCache struct {
+	Cache
+	Expiration time.Duration
+	LoadFunc   func(ctx context.Context, key string) (any, error)
+}
+
+// NewReadThroughCache create readThroughCache
+func NewReadThroughCache(cache Cache, expiration time.Duration,
```

Comment:
How about using a flag to indicate if the GetMulti will reuse the LoadFunction? looks like:
```go
if c.applyForGetMulti {
    // call load func
}
```
And the default value is false. I want to make the users to pay more attention to the GetMulti function.
They must call the NewReadThroughCache(..., ApplyForGetMulti), so they can understand we will call the LoadFunc by iterating the keys.

ApplyForGetMulti should be option pattern.